test/test-lib-nodejs.sh

```diff
@@ -446,8 +446,13 @@ function test_nodemon_present() {
 
 function test_npm_cache_cleared() {
   # Test that the npm cache has been cleared
-  devdep=$(docker run --rm ${IMAGE_NAME}-testapp /bin/bash -c "! test -d \$(npm config get cache)")
-  check_result "$?"
+  # Do not run test on CentOS Stream 9.
```

Comment:
Why not run tests for c9s?

Reply:
Ran into this when checking RHEL9 test failures. This is actually caused by the npm command creating a `.npm/_logs` directory every time npm is called. Skipping this test is actually masking an issue with our assemble script as the cleanup cache step is not the last step in that script (hence we ship the `_logs` directory as well).
I will follow up with a proper fix once I think of a way to work around it.